pandas/io/formats/style.py

```diff
@@ -763,6 +784,41 @@ def set_table_styles(self, table_styles):
         self.table_styles = table_styles
         return self
 
+    def hide_index(self):
+        """
+        Hide any indices from rendering.
+
+        .. versionadded:: 0.20.2
+
+        Returns
+        -------
+        self : Styler
+        """
+        self.index_visible = False
+        return self
+
+    def hide_columns(self, subset):
+        """
+        Hide columns from rendering.
+
```

Comment:
The idea is that you may have columns that are needed for formatting -- determining the color of a row, for example -- but you may not want to display them.  For example, lets say you had a dataframe with columns City and AvgTemp.  Avg Temp was used by Styler to color each row.  But in the final rendering, you only want to display city (colored appropriately).  AFAIK there is no way to do that right now, but with this change you could do df. hide_columns(['AvgTemp']).  You only want to hide AvgTemp.